spec/plugin_spec.rb

```diff
@@ -20,28 +20,26 @@
     subject
   end
 
-  def api_post(url, params = {})
+  def api_post(url, params = {}, expect_status: 200)
```

Comment:
Yes, it's slightly weird. Dropped the default value for `params`, maybe it behaves slightly better now.